drivers/SmartThings/zwave-lock/src/init.lua

```diff
@@ -103,6 +103,9 @@ end
 
 local init_handler = function(driver, device, event)
   populate_state_from_data(device)
+  -- temp fix before this can be changed from being persisted in memory
+  print(utils.stringify_table(constants))
```

Comment:
Did you intend to leave the print statement?

Reply:
definitely not!